### PR TITLE
Ontology environment

### PR DESCRIPTION
**Pull request by MircoNierenz**
# Description

includes some update as discussed in the meeting

## Checklist

- [x] My code follows the modelling guidelines of this project
- [x] I have performed a self-review of my own changes

### CODE REVIEW

File: environment-model/environment-model_ontology.ttl [hunk 1]
```diff
@@ -4,14 +4,15 @@
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
```
Comment (robertschubert) [inline environment-model/environment-model_ontology.ttl:7]: environment-model_instance.json is missing.

File: environment-model/environment-model_ontology.ttl [hunk 2]
```diff
@@ -4,14 +4,15 @@
...
     rdfs:label "ontology definition for environment-model"@en ;
     dcterms:contributor "Mirco Nierenz (TG)" ;
-    owl:versionInfo "0.1"^^xsd:float .
+    owl:versionInfo "0.1" .
 
-environment-model:Asset a owl:Class ;
-    rdfs:label "environment-model" ;
-    rdfs:comment "attributes for 3d environment models"@en ;
-    rdfs:subClassOf gax-core:Resource .
+environment-model:environmentModel a owl:Class ;
```
Comment (robertschubert) [inline environment-model/environment-model_ontology.ttl:14]: class names in PascalCase:
--> EnvironmentModel

File: environment-model/environment-model_shacl.ttl [hunk 3]
```diff
@@ -1,136 +1,159 @@
...
-        [ skos:example "512m" ;
-            sh:datatype xsd:float ;
-            sh:description "tile size in meter"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
-            sh:message "Validation of tile_size failed!"@en ;
-            sh:minCount "0"^^xsd:unsignedInt ;
-            sh:name "tile_size"@en ;
-            sh:path environment-model:environment-model_quantity_tile_size ],
-        [ skos:example "DataSmith" ;
+environment-model:environmentModelShape a sh:NodeShape ;
```
Comment (robertschubert) [inline environment-model/environment-model_shacl.ttl:8]: Classnames in PascalCase
--> EnvironmentModelShape

File: environment-model/environment-model_shacl.ttl [hunk 4]
```diff
@@ -1,136 +1,159 @@
...
+            sh:minCount 1 ;
+            sh:node environment-model:FormatShape ;
+            sh:order 5 ;
+            sh:path environment-model:format ],
+        [ sh:maxCount 1 ;
+            sh:minCount 0 ;            
+            sh:node environment-model:GeoreferenceShape ;
+            sh:order 6 ;
+            sh:path georeference:georeference ] ;
+    sh:targetClass environment-model:environmentModel .
```
Comment (robertschubert) [inline environment-model/environment-model_shacl.ttl:39]: change this target class after ontology change

File: environment-model/environment-model_shacl.ttl [hunk 5]
```diff
@@ -1,136 +1,159 @@
...
+            sh:order 4 ;
+            sh:path environment-model:project ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node environment-model:FormatShape ;
+            sh:order 5 ;
+            sh:path environment-model:format ],
+        [ sh:maxCount 1 ;
+            sh:minCount 0 ;            
+            sh:node environment-model:GeoreferenceShape ;
```
Comment (robertschubert) [inline environment-model/environment-model_shacl.ttl:36]: should be georeference:GeoreferenceShape, right?

File: environment-model/environment-model_shacl.ttl [hunk 6]
```diff
@@ -1,136 +1,159 @@
...
-            sh:datatype xsd:string ;
-            sh:description "name of software vendor"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
-            sh:message "Validation of software_vendor failed!"@en ;
-            sh:minCount "1"^^xsd:unsignedInt ;
-            sh:name "software_vendor"@en ;
-            sh:path environment-model:environment-model_project_software_vendor ],
-        [ skos:example "PBR" ;
+            sh:order 2 ;
+            sh:path environment-model:version ] ;
```
Comment (robertschubert) [inline environment-model/environment-model_shacl.ttl:101]: name clash? environment-model:version already defined.

File: environment-model/environment-model_shacl.ttl [hunk 7]
```diff
@@ -1,136 +1,159 @@
...
+    sh:targetClass environment-model:Project .
+
+environment-model:QualityShape a sh:NodeShape ;
+    sh:property [ skos:example "PBR" ;
             sh:datatype xsd:string ;
-            sh:description "description of quality features"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
+            sh:description "Description of quality features"@en ;
```
Comment (robertschubert) [inline environment-model/environment-model_shacl.ttl:107]: can we add a sh:in here?

File: environment-model/environment-model_shacl.ttl [hunk 8]
```diff
@@ -1,136 +1,159 @@
...
-            sh:minCount "0"^^xsd:unsignedInt ;
+            sh:minCount 0 ;
             sh:name "features"@en ;
-            sh:path environment-model:environment-model_quality_features ],
-        [ skos:example "2" ;
+            sh:order 2 ;
+            sh:path environment-model:features ],
+        [ skos:example "high" ;
+            sh:datatype xsd:string ;
+            sh:description "Category of the level of detail (High - highest level of detail with additional object enrichment, Med - directly from data sources, with environment, Low - topological representation)."@en ;
```
Comment (robertschubert) [inline environment-model/environment-model_shacl.ttl:116]: can we have a sh:in here?

**Review by robertschubert — CHANGES_REQUESTED** [review]

File: environment-model/environment-model_shacl.ttl (hunk 3 above)
Comment (MircoNierenz) [reply environment-model/environment-model_shacl.ttl:8]: fixed

File: environment-model/environment-model_shacl.ttl (hunk 4 above)
Comment (MircoNierenz) [reply environment-model/environment-model_shacl.ttl:39]: fixed

File: environment-model/environment-model_shacl.ttl (hunk 5 above)
Comment (MircoNierenz) [reply environment-model/environment-model_shacl.ttl:36]: fixed

File: environment-model/environment-model_shacl.ttl (hunk 6 above)
Comment (MircoNierenz) [reply environment-model/environment-model_shacl.ttl:101]: change other property to softwareVersion

File: environment-model/environment-model_shacl.ttl (hunk 8 above)
Comment (MircoNierenz) [reply environment-model/environment-model_shacl.ttl:116]: done!

File: environment-model/environment-model_ontology.ttl (hunk 1 above)
Comment (MircoNierenz) [reply environment-model/environment-model_ontology.ttl:7]: add environment-model-instance.json. done!

**Comment by robertschubert** [comment]
@MircoNierenz 
You wrote in https://github.com/GAIA-X4PLC-AAD/ontology-management-base/pull/31 that this one is obsolete.
Since you are the fuction owner please close this PR.

**Pull request closed without merging**